### ISSUE TITLE
Multi Selection in Economy options window by dragging not working

### ISSUE THREAD

**Issue by hessenfarmer** [widelands/widelands#6155]
**Describe the bug**
in previous versions it was possible to select multiple wares in the economy option window by dragging the mouse over them while mousebutton pressed. Even selection by spanning a virtual window was possible. 
currentl this does not work anymore and you need to select all wares by a single click.   

**To reproduce**
1. open economy options 
2. press mouse button over first weapon (top right corner)
3. pull mouse down while button pressed

**Expected behavior**
restore multiselection 
.
